rca/static_src/sass/components/_link.scss

```diff
@@ -242,6 +242,19 @@
         max-width: 170px;
     }
 
+    &--block {
+        color: $color--tertiary !important;
```

Comment:
Hey @patrickcuagan can we avoid the `!important`s?

I wonder if we need this to work against both light and dark backgrounds, too?

Reply:
On guide, editorial, and event pages, the streamfield body has a light background so there's no need to support for dark backgrounds.

As for the `!important` -- this looks to be needed as the streamfield body sometimes (for guide and editorial) is enclosed with `rich-text`. 

```
<div class="streamfield rich-text layout__start-one layout__span-two layout__@large-start-two layout__@large-span-three">
                        {% for block in page.body %}
                            {% include_block block %}
                        {% endfor %}
                    </div>
```

Think removing the `.rich-text` from the parent div is risky. Not sure what other alternatives there are aside from adding the `important`.

Reply:
@patrickcuagan ah right I see. If there's no way around it then perhaps we need to add an ignore for the linter at least?

Reply:
Don't think we have a linter set up. There's quite a lot of !important throughout the build.